pre-commit/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM python:3.7-alpine
+
+RUN apk add --no-cache git npm bash build-base && \
+    npm config set unsafe-perm true && \
+    pip install pre-commit
```

Comment:
Shouldn't this be pinned? Otherwise it's totally unclear which version are we providing...

Reply:
It should. Done. Also set image version to pre-commit version.